Documentation for position, rotation, scaling (#13)
cgi-studio-gmbh/incari-doc: toolbox/incari/object/set-position.md

# Overview

![The Set Position Node.](../../../.gitbook/assets/toolbox/incari/object/set-position.PNG)

> **Set Position** sets the `Position` of an **Object** in a **Scene**.

Reviewer: ```suggestion
**Set Position** sets the value of a given **Object**'s `Position` **Attribute**.
```

# Attributes

|Attribute|Type|Description|
|---|---|---|

> |`Object`|**ObjectID**|The **Object** whose *position* is to be set in the **Scene**, if one is not provided in the `object ID` **Socket**.|

Reviewer: ```suggestion
|`Object`|**ObjectID**|The ID of the **Object** that you would like to set the `Position` value of if one is not provided in the `Object ID` **Socket**.|
```

> |`Default Position`|**Vector3**|The default value of the `Position` **Socket**, if no value is provided.|

Reviewer: ```suggestion
|`Position`|**Vector3**|The desired `Position` value of the **Object** if one is not provided in the `Position` **Socket**.|
```

# Inputs

|Input|Type|Description|
|---|---|---|
|*Pulse Input* (►)|**Pulse**|A standard **Input Pulse**, to trigger the execution of the **Node**.|

> |`Object`|**ObjectID**|The ID of the **Object** you would like to `Set` a `Position` value.|

Reviewer: ```suggestion
|`Object`|**ObjectID**|The ID of the **Object** that you would like to set the `Position` value of.|
```

> |`Position`|**Vector3**|The `Position` you would like to set for the **Object** in a **Scene**.|

Reviewer: ```suggestion
|`Position`|**Vector3**|The desired `Position` value of the **Object**.|
```
